src/pages/home/report/ReportDetailsShareCodePage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import reportPropTypes from '../../reportPropTypes';
+import ONYXKEYS from '../../../ONYXKEYS';
+import ShareCodePage from '../../ShareCodePage';
+
+const propTypes = {
+    /** Navigation route context info provided by react navigation */
+    route: PropTypes.shape({
+        /** Route specific parameters used on this screen */
+        params: PropTypes.shape({
+            reportID: PropTypes.string,
+        }).isRequired,
+    }).isRequired,
+
+    /** The report currently being looked at */
+    report: reportPropTypes,
+};
+
+const defaultProps = {
+    report: undefined,
+};
+
+// eslint-disable-next-line react/prefer-stateless-function
+class ReportDetailsShareCodePage extends React.Component {
+    render() {
+        // eslint-disable-next-line es/no-optional-chaining
```

Comment:
Original line no longer present? 😅

Reply:
Yes, removed that...

Reply:
Do we allow functional components atm? Asking because eslint's `react/prefer-stateless-function` rule  hints me to remove the class component in this case

Reply:
Yess I believe so as we are migrating things to functional components at the moment: https://github.com/Expensify/App/issues/16263

Reply:
Great. thanks!